python/pyspark/worker.py

```diff
@@ -976,6 +977,8 @@ def use_large_var_types(runner_conf):
 # ensure the UDTF is valid. This function also prepares a mapper function for applying
 # the UDTF logic to input rows.
 def read_udtf(pickleSer, infile, eval_type):
+    prefers_large_var_types = False
```

Comment:
What is this larger_var_types?

Reply:
It's https://github.com/apache/spark/pull/41569. Users can set `spark.sql.execution.arrow.useLargeVarTypes` to store large strings or binaries over 2GB in single value